src/content-cli-push.ts

```diff
@@ -139,12 +139,13 @@ class Push {
         program
             .command("package")
             .description("Command to push a package to Studio")
+            .option("--spaceId <spaceId>", "Id of the destination space")
```

Comment:
I'd call this spaceKey. Wouldn't prefer to introduce a new terminology to the end users. They are familiar with keys for packages, and we also show it as `Key` in menu of the Spaces for the same reason